views/default/form_Element_OphInBiometry_Selection_fields.php

```diff
@@ -2,7 +2,15 @@
 <div class="element-fields">
 	<div class="row">
 		<div class="large-12 column">
-			<?php echo $form->dropDownList($element, 'lens_id_'.$side, CHtml::listData(OphInBiometry_LensType_Lens::model()->activeOrPk($element->{'lens_id_'.$side})->findAll(array('order'=> 'display_order asc')),'id','name'),array('empty'=>'- Please select -'),null,array('label'=>3, 'field'=>6))?>
+			<?php
+			$post = Yii::app()->request->getPost('Element_OphInBiometry_Selection');
```

Comment:
1) Business Logic should be in Controller, not in View. 
2) Remove this code base, As it is not required.  It insert default value 0 in db , if lens is not selected .